Resources/config/services.yml

```diff
@@ -1,14 +1,22 @@
+parameters:
+    pim_data_generator.product.product_value_raw_builder.class:  'Pim\Bundle\DataGeneratorBundle\Generator\Product\ProductValueRawBuilder'
```

Comment:
totally useless imvho
in these kind of tools, we should stick to Symfony best practices